db/seeds.rb

```diff
@@ -5,25 +5,26 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+CompanyValue.destroy_all
+company_values = CompanyValue.create!([{title: 'Honesty'}, {title: 'Ownership'}, {title: 'Accountability'}, {title: 'Passion'}])
 employees = Employee.create!([{email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'},{email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'},{email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'},{email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'},{email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'}])
 kudos = Kudo.create!([{
   title: Faker::Adjective.positive,
   content: Faker::Company.bs,
   giver: Employee.create!(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'),
-  receiver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password')
-}, {
+  receiver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'), company_value_id: 'Honesty'},
+   {
   title: Faker::Adjective.positive,
   content: Faker::Company.bs,
   giver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'),
-  receiver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password')
-}, {
+  receiver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'),company_value_id: 'Passion'},
```

Comment:
```suggestion
  receiver: Employee.create(email: Faker::Internet.email(domain: 'gmail.com'), password: 'password'),
  company_value_id: CompanyValue.find_by(title: 'Passion').id
  },
```
You should either pass id or add it by setting `company_value` and not `company_value_id`

Reply:
as Gludek said, you have to pass an instance of `CompanyValue` or id for a given company value. I'm pretty sure that these seeds are not working right now, but I'm not able to check this right now. But you can try to do `rails db:seed:replant` and give us an answer if this creates these two kudos?